recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
For a start it will fail if you run it twice (I'm pretty sure). Secondly, it will only run as Administrator and so may fail fairly cryptically if you aren't running in an elevated prompt. Otherwise, no.